google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableBackupIT.java

```diff
@@ -210,8 +213,8 @@ public void deleteBackupTest() throws InterruptedException {
 
   @Test
   public void restoreTableTest() throws InterruptedException, ExecutionException {
-    String backupId = testEnvRule.env().newPrefix();
-    String restoredTableId = testEnvRule.env().newPrefix() + "-restore";
+    String backupId = prefixGenerator.newPrefix();
+    String restoredTableId = prefixGenerator.newPrefix() + "-restore";
```

Comment:
I know this is only in once place (for now), but maybe it could be nicer to do a builder? `newPrefix().append("-restore")`?

Reply:
I actually want to go the other way and remove this usage :)

Reply:
I removed the suffix, ptal